Autopilot/Path Manager/PathManager.h

```diff
@@ -28,6 +28,7 @@
 //Waypoint types
 #define DEFAULT_WAYPOINT 0
 #define PROBE_DROP_WAYPOINT 1
+#define HOLD_WAYPOINT 2
```

Comment:
How would we send a hold waypoint to the plane? Is the code written on the ground station to be able to add a hold waypoint?